src/Symfony/Component/Form/Tests/ChoiceList/Factory/PropertyAccessDecoratorTest.php

```diff
@@ -138,6 +172,28 @@ public function testCreateViewPreferredChoicesAsPropertyPath()
         ));
     }
 
+    /**
+     * @group legacy
+     */
+    public function testCreateViewPreferredChoicesAsPropertyPathWithCallableString()
+    {
+        $list = $this->getMock('Symfony\Component\Form\ChoiceList\ChoiceListInterface');
+
+        $this->decoratedFactory->expects($this->once())
+            ->method('createView')
+            ->with($list, 'end')
```

Comment:
Your tests fail because it should be `->with($list, $this->isInstanceOf('\Closure'))`.